Homework #3.js

```diff
@@ -0,0 +1,36 @@
+//let, if, else, if else//
+let zodiacSign = "";
+if (zodiacSign.length === 0) {
+	zodiacSign = prompt("What's Your Zodiac Sign? (Fire signs only)");
+} 
+	else if (zodiacSign.toLowerCase () === "leo"); 
```

Comment:
No semicolon here. Next time, you can try this on the Chrome console to test it. It would have thrown an error here.